ui/page/channel/view.jsx

```diff
@@ -238,85 +267,71 @@ function ChannelPage(props: Props) {
-              <TruncatedText text={title || (channelName && '@' + channelName)} lines={2} showTooltip />
-              {odyseeMembership && <MembershipBadge membershipName={odyseeMembership} />}
-            </h1>
-            <div className="channel__meta">
-              <Tooltip title={formattedSubCount} followCursor placement="top">
-                <span>
-                  {getChannelSubCountStr(subCount, compactSubCount)}
-                  {Number.isInteger(subCount) ? (
-                    <HelpLink href="https://help.odysee.tv/category-interaction/following/" />
-                  ) : (
-                    '\u00A0'
-                  )}
-                </span>
-              </Tooltip>
-              {channelIsMine && (
-                <>
-                  {pending ? (
-                    <span>{__('Your changes will be live in a few minutes')}</span>
-                  ) : (
-                    <Button
-                      button="alt"
-                      title={__('Edit')}
-                      onClick={() => push(`?${CHANNEL_PAGE.QUERIES.VIEW}=${CHANNEL_PAGE.VIEWS.EDIT}`)}
-                      icon={ICONS.EDIT}
-                      iconSize={18}
-                      disabled={pending}
-                    />
-                  )}
-                </>
+        <header className="channel-cover" style={coverUrl && { backgroundImage: 'url(' + String(coverUrl) + ')' }}>
+          <div className="channel-header-content">
+            <div className="channel__quick-actions">
+              {isMyYouTubeChannel && (
+                <Button
+                  button="alt"
+                  label={__('Claim Your Channel')}
+                  icon={ICONS.YOUTUBE}
+                  navigate={`/$/${PAGES.CHANNELS}`}
+                />
+              )}
+              <JoinMembershipButton uri={uri} />
+              {!channelIsBlackListed && <ClaimShareButton uri={uri} webShareable />}
+              {!(isBlocked || isMuted) && <ClaimSupportButton uri={uri} />}
+              {!(isBlocked || isMuted) && (!channelIsBlackListed || isSubscribed) && (
+                <SubscribeButton uri={permanentUrl} />
               )}
+              <ClaimMenuList uri={claim.permanent_url} inline />
+            </div>
+            <div className="channel__primary-info">
+              <h1 className="channel__title">
+                <TruncatedText text={title || (channelName && '@' + channelName)} lines={2} showTooltip />
+                {odyseeMembership && <MembershipBadge membershipName={odyseeMembership} />}
+              </h1>
+              <div className="channel__meta">
+                <Tooltip title={formattedSubCount} followCursor placement="top">
+                  <span>
+                    {getChannelSubCountStr(subCount, compactSubCount)}
+                    {Number.isInteger(subCount) ? (
+                      <HelpLink href="https://odysee.com/@OdyseeHelp:b/OdyseeBasics:c" />
```

Comment:
was the `<HelpLink />` change intentional?

Reply:
> was the `<HelpLink />` change intentional?

Nope xD